README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
```

Comment:
i only say that if we think the setup for drupal 8 vs drupal 7 would be very different.